rmgpy/data/kinetics/libraryTest.py

```diff
@@ -81,4 +83,31 @@ def testSaveLibrary(self):
             copyRxns = self.database.libraries['eth-oxcopy'].getLibraryReactions()
             self.assertTrue(all([repr(oriRxns[i])==repr(copyRxns[i]) for i in xrange(len(oriRxns))]))
         finally:
-            shutil.rmtree(os.path.join(settings['test_data.directory'], 'testing_database','kinetics','libraries','eth-oxcopy'))
\ No newline at end of file
+            shutil.rmtree(os.path.join(settings['test_data.directory'], 'testing_database','kinetics','libraries','eth-oxcopy'))
+
+    def test_generate_high_p_limit_kinetics(self):
+        """
+        Test that a :class:Arrhenius kinetics object representing the high pressure limit rate
+        is returned from Troe/Lindmann/PDepArrhenius/Chebyshev kinetic classes
+        """
+        libRxns = self.libraries['lib_net'].getLibraryReactions()
+        for rxn in libRxns:
+            self.assertIsNone(rxn.network_kinetics)
+            logging.debug(rxn)
```

Comment:
If this debug statement was intended to be included I think you should make the message more specific